apps/generator/test/test-project/test-project.test.js

```diff
@@ -18,7 +18,7 @@ const version = '0.0.1';
 console.log = jest.fn();
```

Comment:
ok, we are like kids in the dark here

without logs, we are just guessing

please modify this line 18 and replace it with

```suggestion
// Save the original console.log
const originalConsoleLog = console.log;

// Replace console.log with a custom mock function
console.log = jest.fn((...args) => {
  // Call the original function to actually log to the console
  originalConsoleLog(...args);
});
```

this way we are still able to catch and evaluate logs in tests, but the extra stuff is that we are also able to put debug logs in the console and actually review them, what exactly went wrong